rskj-core/src/main/java/co/rsk/rpc/netty/rest/RestUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rpc.netty.rest;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.codec.http.HttpVersion;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+import static io.netty.handler.codec.http.HttpHeaders.Names.CONTENT_LENGTH;
+import static io.netty.handler.codec.http.HttpHeaders.Names.CONTENT_TYPE;
+
+public class RestUtils {
```

Comment:
Since I've already created an abstract class for modules to inherit (due to the dispatcher refactoring) I will use it to include these methods there as you suggested (I personally think that the inheritance way is the most direct way of doing it).

Reply:
Was thinking on adding it to the recently created `RestModule` abstract class, but it seems to me that those utility methods are really short and basic, and at the same time they're not directly related to the modules _per se_, but to the `REST` server in general (they're needed to create responses and keep everything as standard as possible).

So in this case I personally prefer to keep it as it is (testing it is really simple). But for sure I prefer any of the other strategies when working with bigger or not so specific classes.

WDYT?

Reply:
Fair enough, I have a preference for the other approach but as you mentioned, these methods are small. If this class starts growing, maybe we can change the strategy.